tests/phpunit/CRM/Dedupe/BAO/RuleGroupTest.php

```diff
@@ -181,6 +178,31 @@ public function getSupportedFields() {
           'url' => 'Website',
         ],
     ];
+    $contactTypeFields = [
+      'Organization' => [
+        'legal_name' => 'Legal Name',
+        'organization_name' => 'Organization Name',
+        'sic_code' => 'Sic Code',
+      ],
+      'Individual' => [
+        'birth_date' => 'Birth Date',
+        'is_deceased' => 'Deceased',
+        'deceased_date' => 'Deceased Date',
+        'first_name' => 'First Name',
+        'formal_title' => 'Formal Title',
+        'gender_id' => 'Gender ID',
+        'prefix_id' => 'Individual Prefix',
+        'suffix_id' => 'Individual Suffix',
+        'job_title' => 'Job Title',
+        'last_name' => 'Last Name',
+        'middle_name' => 'Middle Name',
+      ],
+      'Household' => [
+        'household_name' => 'Household Name',
+      ],
+    ];
```

Comment:
This metadata already exists in `DAO_Contact::getFields()`. E.g. the `household_name` field has this:
https://github.com/civicrm/civicrm-core/blob/323a49aca2792dc52c7219f0abb76efd74ecf726/CRM/Contact/DAO/Contact.php#L1740

Reply:
@colemanw yeah - but the point of hard-coding it in the test is to make sure it doesn't change at all & this is a static reference to id

Reply:
Ok sure. As long as the live code is reading from the metadata. Is it?

Reply:
@colemanw the live code also does this https://github.com/civicrm/civicrm-core/pull/29631 - which is fortunately tested so I plan to deconstruct that